platform-metrics/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
   implementation("io.prometheus:simpleclient_pushgateway:0.12.0")
   implementation("org.eclipse.jetty:jetty-servlet:9.4.51.v20230217")
   implementation("com.google.guava:guava:32.0.1-jre")
+  implementation("org.hypertrace.core.documentstore:document-store:0.7.37-SNAPSHOT")
```

Comment:
Adding doc store as a dependency to every service feels overkill. Can you find a way to move this to a separate module so we're not adding a dependency on doc store/db drivers for services that aren't otherwise using them.

Reply:
Makes sense. Updated.